plugins/modules/dedicated_server_task_status.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: dedicated_server_task_status
+short_description: Check the status of an OVH server task
```

Comment:
Isn't this already covered by the `plugins/modules/dedicated_server_install_wait.py` module ?

Reply:
The functionnality is the same as `plugins/modules/dedicated_server_install_wait.py`.